pareto/tests/test_utilities.py

```diff
@@ -393,8 +290,173 @@ def test_fix_vars():
 
 
 ############################
-if __name__ == "__main__":
-    test_utilities_wout_quality()
-    test_utilities_w_post_quality()
-    test_utilities_w_discrete_quality()
-    test_fix_vars()
+def test_data_check():
+    # This emulates what the pyomo command-line tools does
+    # Check that MissingDataError is correctly raised
+    # This input sheet is missing: 'Units',
+    # All of tabs ['ProductionPads', 'CompletionsPads', 'ExternalWaterSources'],
+    # and 'PipelineCapexCapacityBased' which is required for PipelineCost.capacity_based:{}
```

Comment:
I phrased it this way because each of those groupings causes separate warnings/errors. It is confusing for a reader, though, so I like your rephrasing.